packages/utils-debugging-protocol-common/src/debugging-protocol-connector.ts

```diff
@@ -754,6 +759,28 @@ export class Connector implements IConnector {
             // TODO: Investigate why some websites trigger this event twice. Iframe?
```

Comment:
Maybe update the comment now that we know when this can be fired more than once?